tests/XamlParserTests/ServiceProviderTests.cs

```diff
@@ -135,11 +142,6 @@ public void TypeDescriptor_Stubs_Are_Somewhat_Usable()
                 return "Value";
             }, null);
         }
-
-        public static void SetAttachedProperty(ServiceProviderTestsClass target, string value)
```

Comment:
Why do we need this change?

Reply:
This is necessary so that there are no unnecessary messages during build. Since xunit thinks this is a test. If you mark this as private, the tests will stop passing. Therefore, I separated it to another class. It looks better to me -- less dependencies

> Warning	xUnit1013	Public method 'SetAttachedProperty' on test class 'ServiceProviderTests' should be marked as a Theory.